### ISSUE TITLE
OPTIMIZER: Implement sync function for the FVS PRE/POST weighted tables

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#227]
An FVS weighted variable table is created when the first weighted variable is created. For example, PRE_FVS_POTFIRE_WEIGHTED is created when the first PRE_FVS_POTFIRE weighted variable is created. If the list of packages or conditions in a project changes after table creation, BioSum does not update the list of conditions/rxpackages in the WEIGHTED table. This could be a problem if Optimizer is used iteratively and the input list of packages and conditions changes.

We will add a synchronize button that syncs the conditions/rxpackages/rx/rxcycle on the WEIGHTED tables with the current FVS PRE/POST tables. This could add or delete rows. The source weighted variable table(s) may no longer be the same as they were during previous optimizer runs. 

TBD if we will recalculate all the weighted values in the table or use a placeholder value (-1) if conditions are added. If we use a placeholder value, the analyst will need to create/calculate a new weighted variable to calculate it for all conditions. The export/import weights function could be used to make this easier. If we recalculate all the weighted variables, the output may no longer match output for existing scenarios.

**Comment by lbross** [comment]
Add a button to the weighted variables menu to sync the fvs weighted variable tables. The sync function will replace all tables in the prepost_fvs_weighted.accdb. The tables will be rebuilt with the variant/packages/values in the FVS out tables when the sync function runs. All weighted variables will be then recalculated for these stands. Create a backup copy of prepost_fvs_weighted.accdb just in case, but only keep the most recent one.
Because the underlying values and list of stands in the FVS pre/post tables may have changed since earlier Optimizer runs, it is possible that the values will be different than in some optimizer_results.accdb. Jeremy and Sara have indicated this is not a problem.

**Comment by jsfried** [comment]
This seems to work as planned. Column order changes when recalc takes place (I think they sort alphabetically). I did notice that the nulls that were there for cycles other than 1 are now replaced with zeros. Theoretically, info contained in cycles other than zero is never used, right? I have not tested with adding new stands because none of my computers have the ability to do that currently owing to lack of ORACLE.

**Comment by lbross** [comment]
Yes. Data contained in cycles other than 1 is never used. I advise leaving this open until we can test with adding new stands since that action is the catalyst for adding this function.
